src/components/entries/FEEL/Feel.js

```diff
@@ -141,21 +143,26 @@ function FeelTextfieldComponent(props) {
     }
   });
 
-  const handlePopupOpen = (type = 'feel') => {
-    const popupOptions = {
-      id,
-      hostLanguage,
+  const handleExpandProperty = (type = 'feel') => {
```

Comment:
:spiral_notepad: Generalized the terminology here, mostly to align with @nikku's future vision of how we could have similar functionality without breaking established patterns, by actually expanding the property into docked panels, instead of popups.

Expansion is a generic concept that can live in properties panel I believe, popup should not. Hence the direction. Let me know if I'm overreaching here.

Reply:
So the expansion accommodates use cases which don't fit "popup", e.g. showing a separate full screen view. I think this makes sense.